README.md

```diff
@@ -60,229 +32,131 @@ After you're finished please remove all the comments and instructions!
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
-
-> Describe your project in 1 or 2 sentences.
+# 📖 [HELLO-NEW] <a name="about-project"></a>
 
-**[your_project__name]** is a...
+>  this is a design replication task
```

Comment:
- [ ] I think it would be great if you add a description to your project here so add more defintion about what is this project? and what is for? so kindly I recommend to add it to make your README file professional 💯✨😊